manuals/fast_and_fold.md

```diff
@@ -0,0 +1,26 @@
+# Fast and fold!
+AlphaPulldown V2 (beta) now can speed up your modelling process, save your computing time and resources by making multiple input feature matrices have 
+the same shape, as suggested by [AlphaFold](https://github.com/google-deepmind/alphafold#inferencing-many-proteins) themselves. 
+
+For example, using ```pulldown``` mode plus [```baits.txt```](https://github.com/KosinskiLab/AlphaPulldown/blob/main/manuals/example_data/baits.txt) and [```candidates.txt```](https://github.com/KosinskiLab/AlphaPulldown/blob/main/manuals/example_data/candidates.txt)
+will inform AlphaPulldown to generate 294 different modelling tasks and reserve 294 GPU cards to finish them. However, in AlphaPulldown V2, 
```

Comment:
Not sure if "reserve 294 GPU cards" is the correct wording as it might be as well 294 jobs on lower number of cards ;-) Just remove "and reserve 294 GPU cards" part ?